admin/src/app/Components/authorization/authorization.component.html

```diff
@@ -0,0 +1,14 @@
+<div mat-dialog-content>
+    <p>Enter your email</p>
+    <mat-form-field appearance="fill">
+        <mat-label>email</mat-label>
+        <input type="email" matInput placeholder="user@gmail.com" #email>
```

Comment:
I would advise you to use the following code style for html attributes (it's fine to keep 1-2 attributes on the same line, but for the case with 2+ attributes try to brake into lines)

```suggestion
        <input
            type="email"
            matInput
            placeholder="user@gmail.com"
            #email
        >
```

Reply:
Here's one more point.

I would suggest to use official Angular style guide for attributes order.

Order:
1) Structural directives (e.g. `*ngIf`)
2) Template variables (e.g. `#email`)
3) Directives (e.g. `matInput`)
4) Css classes (e.g. `class="name"`)
5) Property binding (e.g. `[name]="name"`)
6) Event binding (e.g. `(change)="OnChange()"`)